packages/calypso-products/src/is-marketplace-product.ts

```diff
@@ -1,8 +1,8 @@
 /**
  * Internal dependencies
  */
-import { YOAST_SEO } from './constants';
+import { MARKETPLACE_PRODUCTS } from './constants';
 
 export function isMarketplaceProduct( product: { productSlug: string } ): boolean {
-	return product.productSlug === YOAST_SEO;
+	return MARKETPLACE_PRODUCTS.includes( product.productSlug );
```

Comment:
Could you remind me what is the difference between this `isMarketplaceProduct()` and the `isMarketplacePlugin()` above and why we need the both? I sort of recall this has been discussed somewhere.

Reply:
Yeah we don't need isMarketplacePlugin I will go ahead and delete it. (Thought it might be useful for something but at this point looks like YAGNI.)